src/Plugin/views/field/OgMembershipBulkForm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\views\field;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\OgMembershipInterface;
+use Drupal\system\Plugin\views\field\BulkForm;
+
+/**
+ * Defines a user operations bulk form element.
+ *
+ * @ViewsField("og_membership_bulk_form")
+ */
+class OgMembershipBulkForm extends BulkForm {
+
+  /**
+   * {@inheritdoc}
+   *
+   * Provide a more useful title to improve the accessibility.
+   */
+  public function viewsForm(&$form, FormStateInterface $form_state) {
+    // Remove all actions related to (non-)member role.
+    foreach ($this->actions as $key => $action) {
+      if (strstr($key, '-member') !== FALSE) {
```

Comment:
Thinking more about this, the check for the default member role should happen in the actions themselves. Non-member is needed.

I am working on actions for the actual membership and they apply to all members (e.g. approve membership, block or even remove membership).